code-quality.gradle

```diff
@@ -77,10 +77,11 @@ jacocoTestReport {
 
 cpdCheck {
     source = sourceSets.main.allJava
-//    reports {
-//        text.enabled = true
-//        xml.enabled = false
-//    }
+    minimumTokenCount = 100
+/*    reports {
```

Comment:
me too, but in this case you can only have either XML or TEXT. Since looking at the XML isn't useful for a human when there is an error I kept it there for easy reference. The XML generated is used by some of our CI (Jenkins) so that is why it isn't always text